addon/components/changeset-input.hbs

```diff
@@ -0,0 +1,5 @@
+<Input
+    @value={{changeset-get @changeset @path}}
+    ...attributes
+    {{on 'input' (event-value (changeset-set @changeset @path))}}
+/>
```

Comment:
since theres no two-way binding allowed here, I dont think theres any benefit of having the extra overhead of using ember's `<Input>` component, unless you can think of anything? otherwise, might be better off just as a native `<input>`:
```hbs
<input
    value={{changeset-get @changeset @path}}
    ...attributes
    {{on 'input' (event-value (changeset-set @changeset @path))}}
/>
``